pom.xml

```diff
@@ -54,7 +54,6 @@
 
     <properties>
         <confluent.maven.repo>http://packages.confluent.io/maven/</confluent.maven.repo>
-        <hadoop.version>2.7.3</hadoop.version>
```

Comment:
@levzem removing this will cause the common version to be used right? Is the version brought in from there identical? Trying to be careful it's not hadoop v 3.x.

Reply:
thats the point of removing it. the version is `2.10.1` so we should be fine and I tested the changes